zcl_text2tab_parser.clas.abap

```diff
@@ -273,6 +301,20 @@ method GET_SAFE_STRUC_DESCR.
 endmethod.  "get_safe_struc_descr
 
 
+  method IS_COMMENT_LINE.
+    DATA: offset TYPE offset.
+
+    is = abap_false.
+
+    FIND FIRST OCCURRENCE OF REGEX '[^[:space:]]' IN line
```

Comment:
I'd insist on checking strictly the first char of the string. There is no indentation in text data so it does not make sense to skip space. `*` can be a part of another field value